### PR TITLE
Add test to workflows steps

### PR DESCRIPTION
**Pull request by LimKianAn**
`make test` is now in *github workflows*.

### CODE REVIEW

**Comment by LimKianAn** [comment]
Debugging ...

**Comment by LimKianAn** [comment]
Now it's working.

File: Makefile [hunk 1]
```diff
@@ -184,11 +184,14 @@ install-configmap-sidecars:
 crd-cwnp-for-testing:
 	curl https://raw.githubusercontent.com/metal-stack/firewall-controller/master/config/crd/bases/metal-stack.io_clusterwidenetworkpolicies.yaml -o external/test/crd-clusterwidenetworkpolicy.yaml
 
+KUBEBUILDER_VERSION:=2.3.1
```
Comment (majst01) [inline Makefile:187]: 2.3.2 is available

**Comment by majst01** [comment]
You develop with go-1.16 but ci is with 1.15 therefore the ci fails with:
```
vet: controllers/suite_test.go:157:20: ReadFile not declared by package os
```

File: .github/workflows/pull_request.yaml [hunk 2]
```diff
@@ -12,6 +12,11 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
+    - name: Ensure Go Version
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16.2'
```
Comment (majst01) [inline .github/workflows/pull_request.yaml:18]: maybe to strict, i would only specify major.minor

**Comment by majst01** [comment]
Super !

**Comment by LimKianAn** [comment]
@eberlep The makefile target *kubebuiler* lets us enjoy the cache check both locally and on github runner. The only drawback is we don't have the github GUI for the skipped step anymore. `make kubebuilder` will always do something in the eyes of github runner.

**Review by eberlep — APPROVED** [review]

**Pull request merged**